<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="50" started-at="2021-01-25T11:46:57 CST" finished-at="2021-01-25T11:46:57 CST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="50" started-at="2021-01-25T11:46:57 CST" finished-at="2021-01-25T11:46:57 CST">
      <class name="learnJavaForInterviews.LearningTestNG">
        <test-method status="FAIL" signature="parameterTest(int, int)[pri:0, instance:learnJavaForInterviews.LearningTestNG@67b92f0a]" name="parameterTest" duration-ms="0" started-at="2021-01-25T11:46:57 CST" finished-at="2021-01-25T11:46:57 CST">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Conversion issue on parameter: value1]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Conversion issue on parameter: value1
at org.testng.internal.Parameters.convertType(Parameters.java:531)
at org.testng.internal.Parameters.createParams(Parameters.java:271)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:356)
at org.testng.internal.Parameters.createParameters(Parameters.java:700)
at org.testng.internal.Parameters.handleParameters(Parameters.java:875)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NumberFormatException: For input string: "param-val-not-found"
at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
at java.lang.Integer.parseInt(Integer.java:580)
at java.lang.Integer.valueOf(Integer.java:766)
at org.testng.internal.Parameters.convertType(Parameters.java:504)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- parameterTest -->
      </class> <!-- learnJavaForInterviews.LearningTestNG -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
